src/Antd_Input.re

```diff
@@ -5,7 +5,7 @@
 [@bs.deriving abstract]
 type jsProps = {
   [@bs.optional] [@bs.as "type"] htmlType: string,
-  [@bs.optional] [@bs.as "name"] htmlName: string,
+  [@bs.optional] [@bs.as "name"] name: string,
```

Comment:
Nice one, let's remove `[@bs.as "name"]` and then I merge

Reply:
updated